charts/firefly/templates/_helpers.tpl

```diff
@@ -274,11 +274,11 @@ database:
     migrations:
       auto: {{ .Values.config.postgresAutomigrate }}
 {{- end }}
-{{- if .Values.config.publicstorageOverride }}
-publicstorage:
-  {{- tpl .Values.config.publicstorageOverride . | nindent 2 }}
+{{- if .Values.config.sharedstorageOverride }}
```

Comment:
This is a breaking change but we added another (#26) as part of the 0.4.0 release and can provide a migration guide if necessary but our adoption of this chart is still low we will not prioritize providing such a guide for now.